analyses/SC_cd_2010/doc_SC_cd_2010.md

```diff
@@ -0,0 +1,23 @@
+# 2010 South Carolina Congressional Districts
+
+## Redistricting requirements
+In South Carolina, districts must:
+
+1. be contiguous
+1. have equal populations
+1. be geographically compact
+1. preserve county and municipality boundaries as much as possible
+
+
+### Interpretation of requirements
+We enforce a maximum population deviation of X.X%.
+
+## Data Sources
+Data for South Carolina comes from the ALARM Project's [2020 Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans for South Carolina.
```

Comment:
This will be the value of `nsims`* `runs` in the simulation. So in this case, 6000.